android-svc-lib.sh

```diff
@@ -46,6 +53,12 @@ DownloadSourceFiles () {
     echo "$g_repoUrl" > "${l_targetDir}/REPO_URL"
 }
 
+###
+# call a service with argument on a device
```

Comment:
`ConvertServiceCallToShellCommand` doesn't actually `call` a service. It just converts/translates one format into another.